pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -31,7 +31,7 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
-var supportedScoringStrategyTypes = sets.NewString(
+var supportedScoringStrategyTypes = sets.New[string](
```

Comment:
```suggestion
var supportedScoringStrategyTypes = sets.New(
```

`sed` didn't catch this one.

Reply:
oh shit. These should be limited cases only. Let me check.

Reply:
Looks like it was the only case